SwiftWisdom.xcodeproj/project.pbxproj

```diff
@@ -1289,15 +1258,19 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
Are these new entries correct and desirable?

Reply:
I'm not sure how these showed up, but I'm also not sure how to make them go away aside from going in and editing the raw file.